topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md

```diff
@@ -0,0 +1,531 @@
+srt@meta.data$Factor.Value.Genotype<-exp_design.tsv$Factor.Value.genotype.
+```
+
+ Now that we have our almost fully annotated object, we will add one more metadata column: percent mitochondrial (perc.mt). This metadata column will denote what percentage of a cell's feature (gene) expression is mitochondrial. 
+
+```{r}
+srt <- PercentageFeatureSet(srt, 
+                               pattern = "^mt-", 
+                               col.name = "perc.mt")
+```
+
+ For the sake of this data set, and many others, the mitochondrial genes will all be marked with an "mt" as the prefix, so that is how we have asked Seurat's PercentageFeatureSet function to search for mitochondrial genes. With that being said, once you are analyzing your own data, it is highly recommended that you figure out how your data set has labelled mitochondrial genes to ensure that you are calculating the correct percentage--the mt prefix may not always include all mitochondrial genes depending on how your dataset was labelled. 
+
+## QC Plots
+ Now that we have a complete Seurat object, we can begin the filtering process. There will  be a number of ‘cells’ that are actually just empty droplets or low-quality. 
+
+There will also be genes that may be sequencing artifacts or that appear with such low frequency that statistical tools will fail to accurately analyse them. 
+
+This background noise of both cells and genes not only makes it harder to distinguish real biological information from sequencing artifacts, but also makes it computationally difficult to analyse. 
+
+First on our agenda is to filter the matrix to give us cleaner data off which to extract meaningful insight from. It will also allow for faster analysis of your data.
+
+We want to filter our cells, but first we need to know what our data looks like. There are a number of subjective choices to make within scRNA-seq analysis, for instance we now need to make our best informed decisions about where to set our thresholds (more on that soon!). 
+
+We’re going to plot our data a few different ways. Different bioinformaticians might prefer to see the data in different ways, and here we are only generating a few of the plots you can use. Ultimately you need to go with what makes the most sense to you.
+
+So let's generate some QC plots. First off, let's check our dataset for batch effect:
+
+```{r}
+VlnPlot(srt,
+        group.by = "Individual",
+        features = "nCount_RNA", 
+        log = TRUE)
+```
+
+This plot shows us the number of cells split by the individual (mouse) from which the cells came from. Now, depending on your experimental design, batch may be represented by something other than individual--like timepoint or even the wet lab researcher who isolated the cells. 
+> Ideally, we would like to see a relatively even distribution of counts for each individual (or batch) but if there isn’t, fear not, we can regress this variable out in a later step.
```

Comment:
```suggestion

Ideally, we would like to see a relatively even distribution of counts for each individual (or batch) but if there isn’t, fear not, we can regress this variable out in a later step.
```

Reply:
you have a number of these, do you intend for them to be rendered in some special way? Should it be a comment box maybe?

Otherwise I think it makes sense to just remove the `>` that starts an unstyled (in the GTN framework) blockquote. By my reading, in most cases in this tutorial, it would be fine without those blockquotes.